docs/static/plugin-manager.asciidoc

```diff
@@ -112,6 +112,22 @@ bin/logstash-plugin update logstash-input-github <2>
 <1> updates all installed plugins
 <2> updates only the plugin you specify
 
+[discrete]
+[[updating-plugins-major-versions]]
+==== Major-version plugin updates
+
+To avoid introducing breaking changes, the plugin manager updates plugins for which newer minor or patch versions exist by default. 
+If you wish to also include breaking changes, specify `--level=major`.
+
+[source,shell]
+----------------------------------
+bin/logstash-plugin update --level=major <1>
+bin/logstash-plugin update --level=major logstash-input-github <2>
+----------------------------------
+<1> updates all installed plugins to latest, including major-version breaking changes
+<2> updates only the plugin you specify to latest, including major-version breaking changes
```

Comment:
```suggestion
[discrete]
[[updating-major]]
==== Major version plugin updates

To avoid introducing breaking changes, the plugin manager updates only plugins for which newer _minor_ or _patch_ versions exist by default.
If you wish to also include breaking changes, specify `--level=major`.

[source,shell]
----------------------------------
bin/logstash-plugin update --level=major <1>
bin/logstash-plugin update --level=major logstash-input-github <2>
----------------------------------
<1> updates all installed plugins to latest, including major versions with breaking changes
<2> updates only the plugin you specify to latest, including major versions with breaking changes
```